cola/icons.py

```diff
@@ -31,9 +32,14 @@
     '.cxx':     'file-code.svg',
 }
 
+cfg = gitcfg.current()
+style = cfg.icon_style()
 
 def install():
-    icon_dir = resources.icon_dir()
+    if not style or style == "light":
+        icon_dir = resources.icon_dir()
+    else:
+        icon_dir = resources.icon_dir(style)
```

Comment:
Just following up -- not sure if you noticed but I ended up keeping the config-driven approach, and ended up augmenting it with `--icon-theme=<theme>` as well as allowing `GIT_COLA_ICON_THEME` in the environment.  The only other minor difference is that the config variable is named `cola.icontheme` whereas it was originally `cola.iconstyle`.